core/src/main/java/org/renjin/primitives/text/StrSignIf.java

```diff
@@ -28,11 +29,14 @@
 
 public class StrSignIf {
 
+  @Internal("formatC")
```

Comment:
The annotation `@Internal` is sufficient if the name is the same as the Java method. Providing the name in the annotation is only necessary if the R name is not a valid Java identifier like "as.double"